api/cart.mjs

```diff
@@ -0,0 +1,79 @@
+import Postmates from "../services/Postmates.mjs";
+import { HTTPResponseError } from "../errors/http.mjs";
+
+/* Search postmates
+ * @param {item} postmates item{} used to initailize cart
+ * @return {object} either success json or error
+ */
+const createPostmatesCart = async (item, itemQuantity) => {
+  try {
+    const postmates = new Postmates();
+    return (await postmates.createCart(item, itemQuantity)).data;
+  } catch (e) {
+    if (e instanceof HTTPResponseError) {
```

Comment:
Since the `createCart` method is already catching this error, there is no need to handle the same error here again. The same applies to the other functions  in this file.

Reply:
Fixed